src/Http/Http.Results/src/Results.cs

```diff
@@ -0,0 +1,1243 @@
+        /// <param name="value">The optional content value to format in the entity body; may be null.</param>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        public static IResult Accepted(Uri uri, object? value)
+        {
+            if (uri == null)
+            {
+                throw new ArgumentNullException(nameof(uri));
+            }
+
+            return new AcceptedResult(locationUri: uri, value: value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <returns>The created <see cref="AcceptedAtRouteResult"/> for the response.</returns>
+        public static IResult AcceptedAtRoute(object? routeValues)
+            => AcceptedAtRoute(routeName: null, routeValues: routeValues, value: null);
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        /// <returns>The created <see cref="AcceptedAtRouteResult"/> for the response.</returns>
+        public static IResult AcceptedAtRoute(string? routeName)
+            => AcceptedAtRoute(routeName, routeValues: null, value: null);
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        ///<param name="routeValues">The route data to use for generating the URL.</param>
+        /// <returns>The created <see cref="AcceptedAtRouteResult"/> for the response.</returns>
+        public static IResult AcceptedAtRoute(string? routeName, object? routeValues)
+            => AcceptedAtRoute(routeName, routeValues, value: null);
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The optional content value to format in the entity body; may be null.</param>
+        /// <returns>The created <see cref="AcceptedAtRouteResult"/> for the response.</returns>
+        public static IResult AcceptedAtRoute(object? routeValues, object? value)
+            => AcceptedAtRoute(routeName: null, routeValues: routeValues, value: value);
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
```

Comment:
Need to swap your 'a' and 'an' around, e.g. ` Creates an <see cref="AcceptedAtRouteResult"/> object that produces a <see cref="StatusCodes.Status202Accepted"/> response`